asdf/asdf.py

```diff
@@ -797,7 +797,7 @@ def _find_asdf_version_in_comments(cls, comments):
         return None
 
     @classmethod
-    def _open_asdf(cls, self, fd, uri=None, mode='r',
```

Comment:
Can `uri` be removed as a parameter?  It's used at the call at https://github.com/asdf-format/asdf/blob/7e63794cd4a8e6b73e33f336dd421edc8b53c8e6/asdf/asdf.py#L950

Reply:
Oops, thanks!  I guess it works due to the `**kwargs`.  I'll remove it from the call and make sure there aren't any other cases.